client/db_test.go

```diff
@@ -32,7 +33,9 @@ import (
 
 func setup() (*server.TestServer, *client.DB) {
 	s := server.StartTestServer(nil)
-	db, err := client.Open("https://root@" + s.ServingAddr() + "?certs=test_certs")
+	db, err := client.Open(fmt.Sprintf("https://%s@%s?certs=test_certs",
```

Comment:
I have a pretty strong preference for the string form. The test will already break if you specify something other than `test_certs`, and grep'ing for `=test_certs` is just as easy as grep'ing for `security.EmbeddedCertsDir`. The only thing that switching to using `security.EmbeddedCertsDir` gets you is that if we do decide to change the directory we have a single place to change, but that seems like a minimal benefit. I have similar feelings about the usage of `security.RootUser`.